phpstan.neon.dist

```diff
@@ -0,0 +1,9 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    reportUnmatchedIgnoredErrors: false
```

Comment:
i would not disable this report, otherwise the baseline could accumulate unneeded exceptions over time.

Reply:
I normally test PHPStan under different PHP and Sf version and keep a `phpstan.neon` file on my filesystem (not comited) to get check if ignored error are still needed, but since the CI is going to test it with only one PHP version and a single related Symfony version it's useless here.